src/commands/appSettings/connectionSettings/netherite/INetheriteConnectionWizardContext.ts

```diff
@@ -18,3 +19,14 @@ export interface INetheriteConnectionWizardContext extends ISetConnectionSetting
 }
 
 export type INetheriteAzureConnectionWizardContext = INetheriteConnectionWizardContext & IEventHubWizardContext;
+
+interface IEventHubWizardContext extends IResourceGroupWizardContext {
```

Comment:
Todo: We should create a common `IEventHubWizardContext` that can be reused by both Netherite and event hub function triggers. 

For now, I think we should keep them separate as the steps can't be completely swapped 1:1 without accidentally messing something up.

Things that will probably need addressing as part of generalizing this in the future:
  - Add a shared folder with the generalized event hub steps
  - `eventhubname` binding setting should be separate from the generalized event hub implementation
  - suggestedLocalSettings for Netherite should no longer be referenced directly in the step (potentially we could explore injecting pick preferences into the general steps)